Adding condor support for sysbatcher

gmadigan opened the pull request:
Nearly identical to its counterpart but obviously submits jobs with condor rather than batch.

DavidMorse commented:
SysBatcher.py already runs on HTCondor, so I'm not sure why a new file is needed here. As far as I can tell only the year and brag arguments were added, which could easily have been added to SysBatcher.py. Am I missing something?

gmadigan commented:
Weird. You're totally right, I must have gotten confused (these are changes I made ages ago that I only now have sorted into a dedicated branch). Did it always use condor? Anyway, I moved the changes to SysBatcher.py and removed the other script, which is reflected in a new commit to this branch.

DavidMorse commented:
It runs on Condor since you updated it last year :)

https://github.com/DavidMorse/LQ2Analysis/commit/d94ebb0c767aa1b023a460bbdea8a9524b994278

DavidMorse commented on SysBatcher.py:
```diff
@@ -20,7 +37,7 @@
 
 # Loop over channels and systematic variations
-for c in ['uujj','uvjj']:
+for c in ['uujj']:
```
please just put back the original line commented out so we remember how to add it back later

gmadigan replied:
Ok, I fixed it. See if that looks good.
